CHANGELOG.mkd

```diff
@@ -3,7 +3,7 @@ CHANGELOG
 
 Unreleased
 ----------
-- Use %LOCALAPPDATA% for Windows cachedir [RK-347](https://tickets.puppetlabs.com/browse/RK-347)
+- Use %LOCALAPPDATA% for Windows cachedir [#1099](https://github.com/puppetlabs/r10k/issues/1049)
```

Comment:
We should just reference the issue. We only link to PRs if there wasn't an associated issue.
```suggestion
- Use %LOCALAPPDATA% for Windows cachedir [#1049](https://github.com/puppetlabs/r10k/issues/1049)
```
I also still think we should include the RK ticket number. @Magisus @justinstoller what do y'all think?

Reply:
I don't think it's necessary. We already communicated to the community that the RK project isn't going to be used anymore in favor of issues. In future if we have more work we want to do for r10k, and there isn't an existing ticket yet, I would imagine we would use the CODEMGMT jira project, which is private, and use an issue to track publicly. I think it's totally reasonable to ignore the Jira ticket here.

Reply:
For reference, @Magisus stated her opinion as
> I think this should only reference the r10k issue and put it in the trailing position like the others.

Reply:
ahh thanks, missed that notification.

okay, we can leave it off then.

Reply:
Agree with Molly though about using the issue, not the PR number.